eos-repartition-mbr

```diff
@@ -140,10 +101,9 @@ udevadm settle
 partprobe "${root_disk}"
 udevadm settle
 
-# mkswap with the new partition index
-swap_partno=$((swap_partno - 2))
-swap_part=${root_part_base}${swap_partno}
-[ -e "$swap_part" ] && mkswap -L eos-swap $swap_part
+# Set marker on 4th partition so that the partition gets enlarged to fill the
+# disk on first boot
+printf "\xdd" | dd of=${root_disk} bs=1 count=1 seek=498 conv=notrunc
```

Comment:
Won't you need a `udevadm settle` after this, in case the `${root_disk}` device is not recreated in time for the scary reads below?